packages/applications-service-api/src/routes/index.js

```diff
@@ -15,6 +15,8 @@ const submissionRouter = require('./submissions');
 const subscriptionRoute = require('./subscriptions');
 const adviceRoute = require('./advice');
 
+router.get('/', (req, res) => res.sendStatus(204));
```

Comment:
should this be router.all() instead?

Reply:
router.get is better since it only matches whereas router.all would match everything (post, delete etc).